Gemfile

```diff
@@ -96,3 +82,23 @@ gem 'ruby-prof-flamegraph'
 gem 'carrierwave', '~> 2.0'
 gem 'carrierwave-mongoid', :require => 'carrierwave/mongoid'
 gem 'uuid'
+
```

Comment:
I don't understand why you reordered these blocks?  development dependencies coming before tests made sense to me

Reply:
Good question -- I reordered because I saw it [docs recommend putting these deps after O{D|R}M-related deps](https://github.com/MiniProfiler/rack-mini-profiler#patching-activerecord).  I've reordered deps in 704f7a8 to satisfy both that and your preference.

Out of curiosity, I surveyed [top Rails repos](https://github.com/topics/rails) and found no clear pattern on this.  [Diaspora](https://github.com/diaspora/diaspora/blob/develop/Gemfile) puts grouped dev/test deps below ungrouped generic deps, but they swap dev/test order.  Putting non-generic deps at bottom also loosely aligns with our JavaScript [`package.json`](https://github.com/broadinstitute/single_cell_portal_core/blob/development/package.json).